packages/flutter/lib/src/widgets/form.dart

```diff
@@ -397,6 +398,23 @@ class FormField<T> extends StatefulWidget {
   /// [FormState.save].
   final FormFieldSetter<T>? onSaved;
 
+  /// An optional property that forces the [FormFieldState] into an error state
+  /// by directly setting the [FormFieldState.errorText] property without
+  /// running the validator function.
+  ///
+  /// When the [forceErrorText] property is provided, the [FormFieldState.errorText]
+  /// will be set to the provided value, causing the form field to be considered
+  /// invalid and display the error message specified.
+  ///
+  /// The [forceErrorText] property is used to override the [InputDecoration.errorText]
+  /// value defined in the [InputDecoration] of the [TextFormField].
+  ///
+  /// Note: Using [forceErrorText] directly to set the error state bypasses the regular
+  /// validation process performed by the [validator] function. This allows you to
+  /// handle error conditions or show specific error messages manually.
+  final String? forceErrorText;
```

Comment:
Nit: What do you think of the name `errorTextOverride` instead? Looking through the docs, I think we use the term "override" in a similar way in a few other places, but I don't see "force" used like that.

Reply:
Or we could even just call it `errorText`, which would match `InputDecoration.errorText`, since they both would do basically the same thing. However, I wonder if people would be confused about the interaction between this and `validator`.

Reply:
I think calling it `errorText` would indeed get people confused, and properly the same goes with `errorTextOverride`.

`errorTextOverride` would get people believe that it will simply override the text messages provided by the `validator`.

However, I'm with you on this one, as I don't really like the name `forceErrorText`, maybe if we make it `forcedErrorText`? the thing is the term `forced` will imply that we will force the error without any negotiation with `validator`.

Whats your take on this?